### ISSUE TITLE
Please make it clear this is an (out of date) unoffical mirror of the code

### ISSUE THREAD

**Issue by PrplHaz4** [apdevstudio/crafty-controller#1]
Per the documentation, the official repo for Crafty Controller is located on Gitlab: https://gitlab.com/crafty-controller/crafty-4

![image](https://github.com/user-attachments/assets/ffd23da0-bc72-42cb-b955-a5f26db328ec)

**Comment by dopreo** [comment]
yeah lol, seems kind of dodgy to me that it still doesn't really seem to say anywhere in this repo that this is unofficial / a mirror
